apps/client/src/components/Conclusion.tsx

```diff
@@ -53,12 +67,12 @@ const Conclusion = ({ checker, matomoTrackEvent }) => {
 
   // Check if the conclusion is 'needContact'
   const contactConclusion = conclusions.find(
-    ({ outcome }) => outcome === sttrOutcomes.NEED_CONTACT
+    (conclusion: string) => conclusion === sttrOutcomes.NEED_CONTACT
```

Comment:
This can't be correct. conclusion is an object.
```
// Check if the conclusion is 'needContact'
  const contactConclusion = conclusions.find(
    ({ outcome }: { outcome: string }) => outcome === sttrOutcomes.NEED_CONTACT
  );
```